blinky-statik/src/main/java/org/spideruci/analysis/statik/flow/IcfgManager.java

```diff
@@ -1,8 +1,10 @@
-package org.spideruci.analysis.statik;
+package org.spideruci.analysis.statik.flow;
```

Comment:
pulling a lot of flow specific code into its own package. i am going to add a new package called dependence that will contain a lot of dependence specific code. I am leaving the Call-graph code in the main `org...analysis.static` package only because I feel it will turn out to be a central data structure based on which a lot of static analysis code will be written.